### PR TITLE
fix: install command sj load

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
This PR fixes problems with loading invalid super.json during install command.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [x] I have read the **CONTRIBUTION_GUIDE** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by TheEdward162 — APPROVED** [review]

File: src/logic/install.ts [hunk 1]
```diff
@@ -609,16 +609,19 @@ export async function installProfiles(
...
   const superJson = loadedResult.match(
     v => v,
     err => {
       logger.error('errorMessage', err.formatLong());
 
-      return new SuperJson({});
+      return undefined;
     }
   );
+  if (!superJson) {
```
Comment (lukas-valenta) [inline src/logic/install.ts:622]: nitpick:
```suggestion
  if (superJson === undefined) {
```

Comment (lukas-valenta) [reply src/logic/install.ts:622]: or maybe rather don't use the match, but just `if (superJson.isErr())` :thinking:

**Review by lukas-valenta — APPROVED** [review]
suggestion: give the PR more descriptive name

File: src/logic/install.ts [hunk 2]
```diff
@@ -611,17 +611,10 @@ export async function installProfiles(
...
-  const superJson = loadedResult.match(
-    v => v,
-    err => {
-      logger.error('errorMessage', err.formatLong());
-
-      return undefined;
-    }
-  );
-  if (!superJson) {
+  if (loadedResult.isErr()) {
```
Comment (lukas-valenta) [inline src/logic/install.ts:614]: suggestion: return the logger.error

Comment (Jakub-Vacek) [reply src/logic/install.ts:614]: Fuu, sorry 😩

**Pull request merged**